pkg/operator/reports.go

```diff
@@ -844,21 +874,31 @@ func isReportNotUsedAsInput(logger log.FieldLogger, report *metering.Report, op
 		if does, depQuery := op.reportQueriesDependOnThisReport(logger, report, reportQueries); does {
 			logger.Infof("ReportQuery, %s exists that uses report %s as input, "+
 				"will not delete though retention period has expired", depQuery, report.Name)
-			return false
+
+			return false, nil
 		}
 	}
-	return true
+	return true, nil
 }
 
+// isReportExpired is a helper function that is responsible for inspecting
+// the @report custom resource and checking whether the user has provided
+// a retention period. In the case that field has been populated, check
+// if the we're currently processing an "expired" report and return true,
+// else return false. We define "expired" here using the following calculation:
+// time.Now() > report.Metadata.CreationTimestamp + report.Spec.Expiration.
 func isReportExpired(logger log.FieldLogger, report *metering.Report, currentTime time.Time) bool {
+	// TODO: should this be contingent on the Report object reporting
+	// a "finished" status?
 	if report.Spec.Expiration != nil {
 		currentTimeNano := currentTime.UTC().Truncate(time.Millisecond).UnixNano()
 		reportCreationNano := report.ObjectMeta.CreationTimestamp.UTC().Truncate(time.Millisecond).UnixNano()
 		retentionAmountNano := report.Spec.Expiration.Nanoseconds()
+
 		if currentTimeNano > (reportCreationNano + retentionAmountNano) {
 			return true
 		}
-		logger.Infof("curr: %d, creat: %d, retain: %d", currentTimeNano, reportCreationNano, retentionAmountNano)
+		logger.Infof("current timestamp: %d, creation timestamp: %d, retention timestamp: %d", currentTimeNano, reportCreationNano, retentionAmountNano)
```

Comment:
```suggestion
		logger.Debugf("current timestamp: %d, creation timestamp: %d, retention timestamp: %d", currentTimeNano, reportCreationNano, retentionAmountNano)
```
Meant to delete this or we can make it debug...